doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -351,27 +443,54 @@ Unary statements are composed of a unary operator (either `+`or `-`) and are use
  
 ## Geographical Queries
 
-Geographical queries are specified by means of the `geometry` and `coords` parameters.
+Geographical queries are specified by means of the `geometry` and `coords` parameters. 
 
 **geometry** defines a geografical area so that only those entities located in such area match the query.
 It is composed of a token list separated by ';'. The first token is the shape of the geometry,
-the rest of tokens (if any) depends on the shape. 
-
-In particular, the following geometries are supported:
-
-+ `geometry=circle;radius:4000`, inside a circle (radius is in meters).
-+ `geometry=circle;radius:4000;external`, outside a circle (radius is in meters).
-+ `geometry=polygon`, inside a polygon.
-+ `geometry=polygon;external`, outside a polygon.
-
-**coords** represents a a list of pairs of geographical coordinates (latitude, longitude) separated by ','.
-Every pair of coordinates represent a geographical point. The list length and its semantics depend on the shape
-specified by the **geometry** parameter:
-
-+ In the case of `circle` geometry, the parameter includes just one element, which
-  is the center of the circle.
-+ In the case of `polygon` includes a list of elements, which are the vertices of the polygon.
-
+the rest of tokens (if any) depends on the shape. The following geometries (see Simple Location Format)
+must be supported:
+
++ `geometry=point`, entities located at exactly that point on the Earth surface.
++ `geometry=line`, entities located along the polygonal line defined.
++ `geometry=circle;radius:4000`, entities inside a circle of the given radius (radius unit must be meters).
++ `geometry=circle;radius:4000;external`, entities outside a circle of the given radius (radius unit must be meters).
++ `geometry=polygon`, entities inside a polygon.
++ `geometry=polygon;external`, entities outside a polygon.
++ `geometry=box`, entities inside a bounding box.
++ `geometry=box;external`, entities outside a bounding box. 
```

Comment:
fixed https://github.com/jmcanterafonseca/fiware-orion/commit/e9b8d2143b4d2008c99c25c53f3e50791b50c2de

wrt to capabilities to the API, I would disagree, due to the fact that introducing optional capabilities is an interoperability hole. Provided that both open source databases (Mongo, PostGIS, etc.) and commercial products support geo-queries as those specified here I think we would not need to introduce something new